script/signup.js

```diff
@@ -67,27 +87,32 @@ function passwordConfirm() {
     return true;
 }
 
-function signUp(event) {
+async function signUp(event) {
     event.preventDefault();
 
     if (
         checkEmail() === true &&
         checkPassword() === true &&
         passwordConfirm() === true
     ) {
```

Comment:
아래와 같이 if 조건문을 ealry return 패턴으로 리팩터링하면 조건문 중첩이 사라지겠네요. 😄 
```suggestion
    if (
        !checkEmail() ||
        !checkPassword() ||
        !passwordConfirm()
    ) {
       return;
    }
```

Reply:
9fd17ef2ce61a3260ec2669e3740009a6393c212 early return 패턴으로 수정하였습니다! 앞으로 조건문 중첩에 대해 더 신경써서 가독성 좋은 코드를 작성할 수 있도록 노력하겠습니다!!